cli/src/commands/mod.rs

```diff
@@ -2853,13 +2853,20 @@ fn cmd_workspace_add(
             .view()
             .get_wc_commit_id(old_workspace_command.workspace_id())
         {
-            tx.repo().store().get_commit(old_wc_commit_id)?.parents()[0].clone()
+            tx.repo().store().get_commit(old_wc_commit_id)?.parents()
```

Comment:
nit: `s/a parent/parents/` in the comment

Reply:
Done.